src/dbnode/storage/database.go

```diff
@@ -1129,3 +1164,24 @@ func (m metadatas) String() (string, error) {
 	buf.WriteRune(']')
 	return buf.String(), nil
 }
+
+func NewAggregateTilesOptions(
```

Comment:
As this is exported type it needs to have a comment in form of `// NewAggregateTilesOptions ...`.

Reply:
👍